containers/src/Kokkos_StaticCrsGraph.hpp

```diff
@@ -21,6 +21,26 @@
 #define KOKKOS_IMPL_PUBLIC_INCLUDE_NOTDEFINED_STATICCRSGRAPH
 #endif
 
+#include <Kokkos_Macros.hpp>
+
+#if defined(KOKKOS_ENABLE_DEPRECATED_CODE_4)
+#if defined(KOKKOS_ENABLE_DEPRECATION_WARNINGS)
+namespace {
+[[deprecated("Deprecated <Kokkos_StaticCrsGraph.hpp> header is included")]] int
+emit_warning_kokkos_static_crs_graph_deprecated() {
+  return 0;
+}
+static auto do_not_include = emit_warning_kokkos_static_crs_graph_deprecated();
+}  // namespace
+#endif
+#else
+#error "Deprecated <Kokkos_StaticCrsGraph.hpp> header is included"
+#endif
+
+#define KOKKOS_IMPL_DEPRECATED_USE_KOKKOS_SPARSE_INSTEAD \
+  KOKKOS_DEPRECATED_WITH_COMMENT(                        \
+      "Moved to KokkosKernels into KokkosSparse namespace")
```

Comment:
```suggestion
      "Moved to KokkosKernels::KokkosSparse namespace")
```
nitpicking but I think it is easier to read

Reply:
No they actually have KokkosSparse as a namespace and I wasn't sure how to convey if you were only using only  Kokkos Core you are out of luck and need to add a dependency on Kernels